new-system-tests/src/test/java/io/apicurio/registry/systemtests/SqlNoIAM.java

```diff
@@ -0,0 +1,117 @@
+        "/apicurio/01_subscription_namespace.yaml", // Apicurio Registry operator subscription in specific namespace
+})
+@TestMethodOrder(MethodOrderer.MethodName.class)
+public class SqlNoIAM extends TestBase {
+    /** {@link OpenShiftClient} instance for tests. */
+    private OpenShiftClient client;
+
+    /**
+     * @return {@link OpenShiftClient} instance used in tests.
+     */
+    @Override
+    protected OpenShiftClient getClient() {
+        return client;
+    }
+
+    /**
+     * Performs actions that should be done before all tests.
+     */
+    @BeforeAll
+    public static void beforeAll() {
+        // Log information about current action
+        System.out.println("Before all.");
+
+        // Client is not accessible here
+    }
+
+    /**
+     * Performs actions that should be done after all tests.
+     */
+    @AfterAll
+    public static void afterAll() {
+        // Log information about current action
+        System.out.println("After all.");
+
+        // Client is not accessible here
+    }
+
+    /**
+     * Performs actions that should be done before each test.
+     */
+    @BeforeEach
+    public void beforeEach() {
+        // Log information about current action
+        System.out.println("Before each.");
+
+        // Get PostgreSQL database service
+        Service svc = client.services().inNamespace(client.getNamespace()).withName(Constants.POSTGRESQL_NAME).get();
+
+        // Get ClusterIP from PostgreSQL database service
+        String clusterIp = svc.getSpec().getClusterIP();
+
+        // Create (or replace) Apicurio Registry instance with PostgreSQL database storage
+        client.resources(ApicurioRegistry.class).resource(buildRegistry(clusterIp)).createOrReplace();
+
+        // Wait for readiness of Apicurio Registry instance with PostgreSQL database storage deployment
+        assertTrue(waitDeploymentReady(client, Constants.REGISTRY_NAME + "-deployment"));
+    }
+
+    /**
+     * Performs actions that should be done after each test.
+     */
+    @AfterEach
+    public void afterEach() {
+        // Log information about current action
+        System.out.println("After each.");
+    }
+
+    /**
+     * Tests that {@link ApicurioRegistry} with PostgreSQL database storage becomes ready.
+     */
+    @Test
+    public void testDeploy() {
+        // Log information about current action
+        System.out.println("### testDeploy test ###");
+
+        // Wait for readiness of PostgreSQL database deployment
+        assertTrue(waitDeploymentReady(client, Constants.POSTGRESQL_NAME));
+
+        // Wait for readiness of Apicurio Registry operator deployment
+        assertTrue(waitDeploymentReady(client, findRegistryOperatorDeployment(client).getMetadata().getName()));
+
+        // Wait for readiness of Apicurio Registry instance with PostgreSQL database storage deployment
+        assertTrue(waitDeploymentReady(client, Constants.REGISTRY_NAME + "-deployment"));
```

Comment:
I already removed it there because I use static YAML definition of Apicurio Registry with static PostgreSQL Service name, so I can check Apicurio Registry instance readiness in test itself (and not in BeforeEach).